lib.php

```diff
@@ -1331,13 +1389,21 @@ function urkund_get_plagiarism_file($cmid, $userid, $file, $relateduserid = null
  * @param int $userid - user id
  * @param varied $file string if path to temp file or full Moodle file object.
  * @param int $relateduserid - related user if if passed. (use when sending to URKUND.
+ * @param string $statusoverride - Override the file's initial queue status. Used for queuing a Mahara HTML Lite download.
  * @return boolean
  */
-function urkund_queue_file($cmid, $userid, $file, $relateduserid = null) {
+function urkund_queue_file($cmid, $userid, $file, $relateduserid = null, $statusoverride = null) {
     global $DB;
     $plagiarismfile = urkund_get_plagiarism_file($cmid, $userid, $file, $relateduserid);
     $plagiarismvalues = $DB->get_records_menu('plagiarism_urkund_config', array('cm' => $cmid), '', 'name, value');
 
+    // Check if $plagiarismfile needs to be downloaded from Mahara.
```

Comment:
this comment doesn't really explain what's going on here... "if statusoveride is passed, update the record with the new value - used by mahara" .. or something along those lines...